package.json

```diff
@@ -26,6 +26,7 @@
     "grid-styled": "4.1.0",
     "history": "3",
     "humanize-plus": "^1.8.1",
+    "husky": "^4.0.0",
```

Comment:
Just out of curiosity, why not using the latest version (6)?

Reply:
Oh, they slightly changed API, updated it to use 6 version. Thanks for pointing out.